### ISSUE TITLE
Revisit warnings from 850 ruleset

### ISSUE THREAD

**Issue by AMDmi3** [repology/repology-rules#194]
A lot of warnings have accumulated from 850 ruleset - additional split rules are needed for these packages.

**Comment by AMDmi3** [comment]
Complete list:

```
Rule warning for allegro (allegro-git) in aur: please classify me
Rule warning for ape (ape) in nix_stable: please classify me
Rule warning for ape (ape) in nix_unstable: please classify me
Rule warning for bcc (bcc) in pld: please classify me
Rule warning for clink (clink) in crux_34: please classify me
Rule warning for clink (clink) in crux_35: please classify me
Rule warning for ctags (ctags) in aix_osp: please classify me
Rule warning for ctags (ctags) in aosc: please classify me
Rule warning for ctags (ctags) in manjaro_unstable: please classify me
Rule warning for ctags (ctags) in ravenports: please classify me
Rule warning for ctags (universal-ctags) in chocolatey: please classify me
Rule warning for ctags (universal-ctags) in freebsd: please classify me
Rule warning for ctags (universal-ctags) in kali_rolling: please classify me
Rule warning for ctags (universal-ctags) in nix_stable: please classify me
Rule warning for ctags (universal-ctags) in nix_unstable: please classify me
Rule warning for ctags (universal-ctags) in openbsd: please classify me
Rule warning for ctags (universal-ctags) in openbsd: please classify me
Rule warning for ctags (universal-ctags) in parrot: please classify me
Rule warning for ctags (universal-ctags) in ubuntu_19_04: please classify me
Rule warning for ctags (universal-ctags) in ubuntu_19_10: please classify me
Rule warning for ctags (universal-ctags-git) in aur: please classify me
Rule warning for db (berkeley-db) in homebrew: please classify me
Rule warning for db (berkeley-db) in linuxbrew: please classify me
Rule warning for db (db) in distrowatch: please classify me
Rule warning for dex (dex) in nix_unstable: please classify me
Rule warning for dex (dex) in openpkg_current: please classify me
Rule warning for ecm (ecm) in chocolatey: please classify me
Rule warning for elm (elm) in openpkg_current: please classify me
Rule warning for fail (fail) in void_x86_64: please classify me
Rule warning for icecream (icecream) in aur: please classify me
Rule warning for icecream (icecream) in homebrew: please classify me
Rule warning for icecream (icecream) in linuxbrew: please classify me
Rule warning for icecream (icecream-git) in aur: please classify me
Rule warning for love (love-hg) in aur: please classify me
Rule warning for love (love6) in scoop: please classify me
Rule warning for mako (mako) in nix_stable: please classify me
Rule warning for mako (mako) in nix_unstable: please classify me
Rule warning for mars (mars) in pld: please classify me
Rule warning for mpc (mpc) in pld: please classify me
Rule warning for mpd (mpd) in kaos: please classify me
Rule warning for mpd (mpd) in pld: please classify me
Rule warning for netcat (netcat) in openpkg_current: please classify me
Rule warning for pal (pal) in chocolatey: please classify me
Rule warning for pgadmin (pgadmin) in aosc: please classify me
Rule warning for python (python2) in freebsd: please classify me
Rule warning for python (python3) in freebsd: please classify me
Rule warning for python (python3.7) in nix_stable: please classify me
Rule warning for python (python3.7) in nix_unstable: please classify me
Rule warning for rdd (rdd) in ubuntu_14_04: please classify me
Rule warning for rdd (rdd) in void_x86_64: please classify me
Rule warning for sample (Sample) in chocolatey: please classify me
Rule warning for sdb (sdb) in openmandriva_cooker: please classify me
Rule warning for swift (swift) in homebrew: please classify me
Rule warning for swift (swift) in linuxbrew: please classify me
Rule warning for swift (swift) in nix_stable: please classify me
Rule warning for swift (swift) in nix_unstable: please classify me
Rule warning for sx (sx-git) in aur: please classify me
Rule warning for tesseract (tesseract) in pld: please classify me
Rule warning for tesseract (tesseract) in scoop: please classify me
Rule warning for words (words) in pld: please classify me
```

**Comment by AMDmi3** [comment]
Updated list:

```
Rule warning for ape (ape) in nix_stable: please classify me
Rule warning for ape (ape) in nix_unstable: please classify me
Rule warning for ctags (ctags) in aix_osp: please classify me
Rule warning for ctags (ctags) in aosc: please classify me
Rule warning for ctags (ctags) in gentoo: please classify me
Rule warning for ctags (ctags) in gentoo: please classify me
Rule warning for ctags (ctags) in gentoo: please classify me
Rule warning for ctags (ctags) in manjaro_unstable: please classify me
Rule warning for ctags (ctags) in ravenports: please classify me
Rule warning for ctags (universal-ctags) in chocolatey: please classify me
Rule warning for ctags (universal-ctags) in freebsd: please classify me
Rule warning for ctags (universal-ctags) in kali_rolling: please classify me
Rule warning for ctags (universal-ctags) in nix_stable: please classify me
Rule warning for ctags (universal-ctags) in nix_unstable: please classify me
Rule warning for ctags (universal-ctags) in openbsd: please classify me
Rule warning for ctags (universal-ctags) in openbsd: please classify me
Rule warning for ctags (universal-ctags) in parrot: please classify me
Rule warning for ctags (universal-ctags) in ubuntu_19_04: please classify me
Rule warning for ctags (universal-ctags) in ubuntu_19_10: please classify me
Rule warning for ctags (universal-ctags-git) in aur: please classify me
Rule warning for db (berkeley-db) in homebrew: please classify me
Rule warning for db (berkeley-db) in linuxbrew: please classify me
Rule warning for db (db) in distrowatch: please classify me
Rule warning for db (db) in gentoo: please classify me
Rule warning for db (db) in gentoo: please classify me
Rule warning for ecm (ecm) in chocolatey: please classify me
Rule warning for elm (elm) in openpkg_current: please classify me
Rule warning for fail (fail) in void_x86_64: please classify me
Rule warning for icecream (icecream) in aur: please classify me
Rule warning for icecream (icecream) in fedora_29: please classify me
Rule warning for icecream (icecream) in gentoo: please classify me
Rule warning for icecream (icecream) in gentoo: please classify me
Rule warning for icecream (icecream) in gentoo: please classify me
Rule warning for icecream (icecream) in gentoo: please classify me
Rule warning for icecream (icecream) in homebrew: please classify me
Rule warning for icecream (icecream) in kali_rolling: please classify me
Rule warning for icecream (icecream) in linuxbrew: please classify me
Rule warning for icecream (icecream) in parrot: please classify me
Rule warning for icecream (icecream) in pld: please classify me
Rule warning for icecream (icecream) in rosa_2016_1: please classify me
Rule warning for icecream (icecream) in trisquel_7_0: please classify me
Rule warning for icecream (icecream) in ubuntu_14_04: please classify me
Rule warning for icecream (icecream) in ubuntu_16_04: please classify me
Rule warning for icecream (icecream) in ubuntu_18_04: please classify me
Rule warning for icecream (icecream) in ubuntu_18_10: please classify me
Rule warning for icecream (icecream) in ubuntu_19_04: please classify me
Rule warning for icecream (icecream) in ubuntu_19_10: please classify me
Rule warning for icecream (icecream) in yacp: please classify me
Rule warning for icecream (icecream-git) in aur: please classify me
Rule warning for jack (jackit) in gentoo: please classify me
Rule warning for love (love) in gentoo: please classify me
Rule warning for love (love-hg) in aur: please classify me
Rule warning for mars (mars) in pld: please classify me
Rule warning for mpc (mpc) in pld: please classify me
Rule warning for mpd (mpd) in kaos: please classify me
Rule warning for mpd (mpd) in pld: please classify me
Rule warning for muscle (muscle) in yacp: please classify me
Rule warning for netcat (netcat) in gentoo: please classify me
Rule warning for netcat (netcat) in gentoo: please classify me
Rule warning for netcat (netcat) in openpkg_current: please classify me
Rule warning for pal (pal) in chocolatey: please classify me
Rule warning for python (python2) in freebsd: please classify me
Rule warning for python (python3) in freebsd: please classify me
Rule warning for python (python3.7) in nix_stable: please classify me
Rule warning for python (python3.7) in nix_unstable: please classify me
Rule warning for sample (Sample) in chocolatey: please classify me
Rule warning for sdb (sdb) in openmandriva_cooker: please classify me
Rule warning for sx (sx-git) in aur: please classify me
Rule warning for tesseract (tesseract) in pld: please classify me
```

**Comment by AMDmi3** [comment]
Done. The only currently remaining warning is `Rule warning for python (python3.7) in nix_unstable: please classify me` caused by ambiguity of nix name/version splitting (see repology/repology#854).
